package.json

```diff
@@ -4,30 +4,36 @@
   "description": "Codemods, public or private, are preferred to reside in repositories owned by the main organization of their respective projects. This repository serves as a catch-all for some codemod sources that still have not found thier way to their ideal home or for Codemod-compatible forks of existing codemods for easier publishing. Please [chat with us](https://go.codemod.com/community) before making any major contribution.",
   "main": "index.js",
   "dependencies": {
-    "prettier": "^3.4.2"
+    "husky": "^9.1.5",
```

Comment:
Why do we have v9 as a normal dependency and v8 as a dev dependency? shouldn't it all be devDependencies?